src/Entity/Consumer.php

```diff
@@ -23,39 +24,45 @@ public function __construct()
     {
         $this->reviews = new ArrayCollection();
     }
-
-    public function __toString()
-    {
-        return $this->consumer_name;
-    }
-
+//    public function __toString()
+//    {
+//        return $this->consumer_name;
+//    }
```

Comment:
Would this comment change anything? Are you still getting exception when __toString metod is removed?

Reply:
Also what about admin panel? Is it still working, since we had some issues about missing `__toString` method? Or maybe we kind a don't need it any more?

Reply:
We need him, I will remove the comment